account_payment_partner/models/account_move.py

```diff
@@ -110,7 +110,8 @@ def get_bank_id():
                         and move.commercial_partner_id.bank_ids
                     ):
                         bank_id = get_bank_id()
-            move.invoice_partner_bank_id = bank_id
+            if bank_id:
+                move.invoice_partner_bank_id = bank_id
```

Comment:
Can you specify the steps of a real case where this condition is needed?

Reply:
Creating an invoice of type "**out_invoice**" the field "**invoice_partner_bank_id**" have a default value res.partner.bank(47,) by example.

If we don't ask for the value before assigning, it replaces it with False.

Before this changes: 

![image](https://user-images.githubusercontent.com/3088246/128058776-0f499177-ed8f-480a-a2fa-6eb49d7f1b74.png)

After:

![image](https://user-images.githubusercontent.com/3088246/128058889-43b922f8-ebad-4fe3-8d72-9ec29a4de4e4.png)

The company's default account has been set by default when creating the invoice.